src/main.c

```diff
@@ -173,7 +223,16 @@ void main(void)
 {
     vector_init(&tests_to_run, kernel_thunk_table_size);
     vector_init(&tests_exclude, 20);
-    if (!open_output_file("D:\\kernel_tests.log")) {
+    load_name_file("D:\\name.txt");
```

Comment:
Instead of going through all this trouble of reading a "name" file just to retrieve a `name_log`, why isn't this just read from the command line arguments? (Or is that feature not available on Xbox - I don't remember that detail...)

Reply:
Xbox is not a desktop, none of the consoles to this day are desktops. So, there is no such thing as command line interface. Internally? Yes, there is arguments support designed from one app to another app. The difference is they are not exposed to users. Unless, of course, the main app is a development tool.

In summary, it will require create such app to pass an argument and read from `LaunchDataPage` export. Plus it's best not to rely on third-party app to supply such info.